sap-service-operator/src/main/java/com/sap/cloud/environment/servicebinding/SapServiceOperatorServiceBindingIoAccessor.java

```diff
@@ -176,6 +188,22 @@ private Path getRootDirectory()
         return rootDirectory;
     }
 
+    @Nullable
+    private Path getFallbackRootDirectory()
+    {
+        if( fallbackBindingRootPath == null ) {
+            return null;
+        }
+
+        logger.debug("Trying to fall back to '{}'.", fallbackBindingRootPath);
+        if( !Files.exists(fallbackBindingRootPath) || !Files.isDirectory(fallbackBindingRootPath) ) {
```

Comment:
I think you can skip the `Files.exists` check, as this is also part of the `Files.isDirectory` check: https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#isDirectory(java.nio.file.Path,%20java.nio.file.LinkOption...)

Reply:
I adjusted the usages accordingly. Thanks for pointing this out.